File: solutions/java15/medium/class-and-objects-sports-venue-information-system-1/solution-approach1.java
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/contests/ku-uit-oops/challenges/class-and-objects-sports-venue-information-system-1/problem?isFullScreen=true
// Problem     Class and Objects - Sports Venue Information System 1
// Difficulty  Medium
// Subdomain   N/A
// Platform    HackerRank
// Language    java15
// Status      Accepted
// Submitted   2026-08-23, 03:27 p.m.
// ──────────────────────────────────────────────────

import java.io.*;
import java.util.*;

class Venue {
    private String name;
    private String city;
    
    Venue(String name, String city) {
        this.name = name;
        this.city = city;
    }
    void displayVenueDetails() {
        System.out.println("Venue Details ");
        System.out.println("Venue Name : " + name);
        System.out.println("City Name : " + city);
    }
}
public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the venue details");
        String input = sc.nextLine();
        String[] details = input.split(",");
        Venue venue = new Venue(details[0].trim(), details[1].trim());
        venue.displayVenueDetails();
        sc.close();
    }
}

File: solutions/cpp/medium/linear-search-in-a-library/solution-approach1.cpp
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/contests/ku-uit-daa/challenges/linear-search-in-a-library/problem?isFullScreen=true
// Problem     Searching: Linear Search in a Library
// Difficulty  Medium
// Subdomain   N/A
// Platform    HackerRank
// Language    cpp
// Status      Accepted
// Submitted   2026-09-17, 01:23 p.m.
// ──────────────────────────────────────────────────

#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    int n;
    if (!(cin >> n)) return 0;

    int arr[n];
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
    }

    int x;
    cin >> x;

    int ans = -1;
    for (int i = 0; i < n; i++) {
        if (arr[i] == x) {
            ans = i;
            break;
        }
    }

    cout << ans << endl;

    return 0;
}
